### DISCUSSION TITLE
Styles Missing After Deploying Atlantis ng Template to OnRender

### DISCUSSION THREAD

**Discussion by hafssaidt** [primefaces/community#2814] [PrimeNG Templates]
i am using **atlantis ng** template it works locally but when I **deploy** it to onrender it doesn't loads the styles. Any idea why this happens or how to fix it?

Thanks

**Comment by cagataycivici** [comment]
Could you please state your Atlantis version?

**Reply by hafssaidt** [reply]
I’m using version 17

**Comment by cagataycivici** [comment]
What is the name of the zip file you used? Is it atlantis-ng-9.0.0.zip?

**Reply by hafssaidt** [reply]
No I used atlantis-ng-17.0.0.zip
